tests/foreman/api/test_syncplan.py

```diff
@@ -167,9 +175,52 @@ def test_create_sync_plan_with_invalid_random_name(self, name):
         @Feature: SyncPlan
 
         """
+        with self.assertRaises(HTTPError):
+            entities.SyncPlan(
+                name=name,
+                organization=self.org,
+            ).create()
 
+    @data(
+        '',
+        ' ',
+        None,
+    )
+    def test_create_sync_plan_with_no_enabled_value_negative(self, enabled):
+        """@Test: Create a sync plan with empty 'enabled' field.
 
-@stubbed()
+        @Assert: A sync plan can not be created with empty 'enabled' field.
+
+        @Feature: SyncPlan
+
+        """
+        with self.assertRaises(HTTPError):
+            entities.SyncPlan(
+                enabled=enabled,
```

Comment:
I'm checking both no value and invalid value here. In both cases sync_plan creation should fail.
What's the advantage of omitting a field in that way? We will have to manually specify some random interval, name, generate some date. It won't decrease code quantity, and, won't increase code readability - imo, if only 'enabled' parameter specified means i'm working only with it; if 5 arguments are specified but one is missing - i'm not sure which one is missing and what i'm actually testing here.
Everything above is just imo, please, share your thoughts on that :)